osu.Game.Rulesets.Osu/OsuDifficulty/Preprocessing/OsuDifficultyHitObject.cs

```diff
@@ -0,0 +1,58 @@
+﻿// Copyright (c) 2007-2017 ppy Pty Ltd <contact@ppy.sh>.
+// Licensed under the MIT Licence - https://raw.githubusercontent.com/ppy/osu/master/LICENCE
+
+using System;
+using osu.Game.Rulesets.Osu.Objects;
+
+namespace osu.Game.Rulesets.Osu.OsuDifficulty.Preprocessing
+{
+    public class OsuDifficultyHitObject
+    {
+        public OsuHitObject BaseObject { get; }
+
+        /// <summary>
+        /// Normalized distance from the StartPosition of the previous note.
+        /// </summary>
+        public double Distance { get; private set; }
+
+        /// <summary>
+        /// Milliseconds elapsed since the StartTime of the previous note.
+        /// </summary>
+        public double Ms { get; private set; }
+
+        public double MsUntilHit { get; set; }

```

Comment:
Same thing here. Also _every_ public or protected member that isn't an override needs an xml-doc explaining what it does / what its purpose is.